src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -973,5 +974,5 @@ else if (CommonUtil.startsWithChar(pkg, '/')) {
         }
         return patterns;
     }
-
```

Comment:
Please restore this removed line. We don't expect any unnecessary changes.

Reply:
done